tests/utils/notebook_test/__init__.py

```diff
@@ -111,7 +111,7 @@ def run_notebook(notebook, notebook_dir, kernel=None, no_cache=False, temp_dir='
             nbformat.write(notebook, output_file)
             output_nb = io.open(output_file, mode='r', encoding='utf-8')
             for line in output_nb:
-                if "Warning:" in line:
+                if "Warning:" in line and "numpy operator signatures" not in line:
```

Comment:
@reminisce

Reply:
The warning message of `numpy operator signature` is for python version < 3.5. It does not affect functionality, but merely used for building numpy op documentation. I'm fine with this change.